### PR TITLE
Fix for number of cores taken from cpu experiments

### PR DESCRIPTION
**Pull request by Cerfoglg**

### CODE REVIEW

File: analysers/experiments/cpu.py [hunk 1]
```diff
@@ -181,6 +181,12 @@ def sortAndGet(field, asc):
 CassandraRDD = sc.cassandraTable(cassandraKeyspace, "trial_cpu_core")
 CassandraRDD.cache()
 
+nOfCores = CassandraRDD.select("cpu_mean") \
```
Comment (VincenzoFerme) [inline analysers/experiments/cpu.py:184]: Why do you need to access the `nOfCores` this way? We should have it in the Cassandra schema already.

Comment (Cerfoglg) [inline analysers/experiments/cpu.py:184]: @VincenzoFerme  There we have the active CPUs, but I want the total number of CPUs

Comment (VincenzoFerme) [inline analysers/experiments/cpu.py:184]: And why?

File: analysers/trials/cpu.py [hunk 2]
```diff
@@ -52,7 +52,7 @@ def computeMetrics(data):
     dataIntegral = integrate.trapz(data).item()
 
     # TODO: Fix this
-    return [{"experiment_id":experimentID, "trial_id":trialID, "container_id":containerID, "cpu_median":median, "cpu_cores":nOfCores, \
+    return [{"experiment_id":experimentID, "trial_id":trialID, "container_id":containerID, "cpu_median":median, "cpu_cores":nOfActiveCores, \
```
Comment (VincenzoFerme) [inline analysers/trials/cpu.py:55]: Where do you define `nOfActiveCores`?

Comment (Cerfoglg) [inline analysers/trials/cpu.py:55]: @VincenzoFerme  It's stored in environment_data, I get it from there

File: analysers/experiments/cpu.py (hunk 1 above)
Comment (Cerfoglg) [inline analysers/experiments/cpu.py:184]: @VincenzoFerme To have the correct indexes for the cores, and avoiding index out of range problems.

**Pull request merged**